sunpy/visualization/colormaps/color_tables.py

```diff
@@ -69,16 +69,16 @@ def paden(lst, len_, pad=0):
 c3 = ((c1 + c2 / 2.0) * 255.0 / (c1.max() + c2.max() / 2.0)).astype('f')
 
 aia_wave_dict = {
-    1600: (c3, c3, c2),
-    1700: (c1, c0, c0),
-    4500: (c0, c0, b0 / 2.0),
-    94: (c2, c3, c0),
-    131: (g0, r0, r0),
-    171: (r0, c0, b0),
-    193: (c1, c0, c2),
-    211: (c1, c0, c3),
-    304: (r0, g0, b0),
-    335: (c2, c0, c1)
+    1600: (c3, c3, c2)*u.angstrom,
+    1700: (c1, c0, c0)*u.angstrom,
+    4500: (c0, c0, b0 / 2.0)*u.angstrom,
+    94: (c2, c3, c0)*u.angstrom,
+    131: (g0, r0, r0)*u.angstrom,
+    171: (r0, c0, b0)*u.angstrom,
+    193: (c1, c0, c2)*u.angstrom,
+    211: (c1, c0, c3)*u.angstrom,
+    304: (r0, g0, b0)*u.angstrom,
+    335: (c2, c0, c1)*u.angstrom
```

Comment:
Ah sorry I should have read the example clearer. You want these on the keys and not the values of the dictionary.